tests/babel_loose_array_spread/babel_loose_array_spread.exp

```diff
@@ -16,6 +16,24 @@ References:
                                     ^^^^^^^^^^^^^^^^^ [2]
 
 
+Error --------------------------------------------------------------------------------------------------- apply.js:11:15
```

Comment:
This is  a new error caused by splitting `$IteratorProtocol` from `Iterator`. It's in addition to the existing errors on this line, which were preserved.